web/html/src/components/formulas/EditGroup.js

```diff
@@ -217,7 +217,9 @@ class EditDictionaryGroup extends React.Component {
         return (
             <div className="form-group" key={element_name}>
                 <label className="col-lg-3 control-label" style={{"color": "green"}}>
```

Comment:
What about removing the `green` color completely here which is not consistent with the rest of the product?

Reply:
I agree with you, I didn't change it because I was assuming it had some relevant value to the customer, but we already have that info in the help icon: "This field is used as a 'key' identifier in the resulting pillar data."... 

I will follow your suggestion but I would like to hear @hustodemon opinion before merging

Reply:
updated screenshot and PR :)